.github/workflows/cts_ci.yml

```diff
@@ -128,6 +128,7 @@ jobs:
     container:
       # image: khronosgroup/sycl-cts-ci:${{ matrix.sycl-impl }}-${{ matrix.version }}
       # ComputeCpp images are hosted by Codeplay
+      # TODO: remove patching ComputeCpp configuration when image gets updated
```

Comment:
Do we still need any changes in this file?

Reply:
True... removed. (waiting for CI to finish before blasting it again with a comment-only change)

Reply:
You shouldn't worry about CI. CI will run tests for all commits, and you will be able to review all results.
BTW, here are results for https://github.com/KhronosGroup/SYCL-CTS/pull/424/commits/a0e720dc28c3126a28de7b6f6e83d01dc07d9443 - https://github.com/KhronosGroup/SYCL-CTS/actions/runs/4116155955/jobs/7105833324. Patched ComputeCPP build fails.

Reply:
Pushed removal.